camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/ReferencedYamlTest.java

```diff
@@ -142,7 +192,7 @@ public void testYamlUrlReferencingCallerCatalogItem() throws Exception {
             "    version: " + TEST_VERSION,
             "    item:",
             "      services:",
-            "      - type: classpath://yaml-ref-parent-catalog.yaml");
+            "      - type: classpath://yaml-ref-catalog.yaml");  // this references yaml.basic above
```

Comment:
Wow, I didn't no we supported this. Why?!

It looks like `classpath://yaml-ref-catalog.yaml` contains:
```
name: Basic app
services:
- name: service
  type: yaml.basic:0.1.2
```
so it's not a catalog item. Should we really be able to reference it as a type in a catalog item like this? I'd have thought the right way of doing it (for an end-user) would be to re-write `yaml-ref-catalog.yaml` as a catalog item, deploy it to the catalog, and then reference its type in the normal way (which clearly would invalidate this test case).

I suggest we deprecate this asap. I don't think we've documented it anywhere, so hopefully no-one is relying on it in anger.

I therefore think it's not worth fixing this test.